### ISSUE TITLE
the accuracy is very low when using bnfused ckpt

### ISSUE THREAD

**Issue by remme123** [ARM-software/ML-KWS-for-MCU#77]
Hi:
The accuracy is very low when I test the ds-cnn model with batch-norm fused ckpt.But the unfused ckpt is quit normal.
**batch-norm:**
`python fold_batchnorm.py --model_architecture ds_cnn --model_size_info 5 64 10 4 2 2 64 3 3 1 1 64 3 3 1 1 64 3 3 1 1 64 3 3 1 1 --dct_coefficient_count 10 --window_size_ms 40 --window_stride_ms 20 --data_url= --wanted_words=yes,no,on,off --checkpoint=work/DS_CNN/DS_CNN_S/training/best/ds_cnn_9631.ckpt-30400`

**unfused test:**
`python test.py --model_architecture ds_cnn --model_size_info 5 64 10 4 2 2 64 3 3 1 1 64 3 3 1 1 64 3 3 1 1 64 3 3 1 1 --dct_coefficient_count 10 --window_size_ms 40 --window_stride_ms 20 --data_url= --wanted_words=yes,no,on,off --checkpoint=work/DS_CNN/DS_CNN_S/training/best/ds_cnn_9631.ckpt-30400`
> 2019-01-03 19:56:24.642860: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA
INFO:tensorflow:Restoring parameters from work/DS_CNN/DS_CNN_S/training/best/ds_cnn_9631.ckpt-30400
INFO:tensorflow:set_size=14898
INFO:tensorflow:Confusion Matrix:
 [[1239    0    0    0    0    0]
 [   0 1219    1   12    0    1]
 [   1    0 3272    5    0    1]
 [   0    0    0 3142    0    0]
 [   0    2    2    3 3023   12]
 [   0    0    3    8    0 2952]]
INFO:tensorflow:Training accuracy = 99.66% (N=14898)
INFO:tensorflow:set_size=1847
INFO:tensorflow:Confusion Matrix:
 [[154   0   0   0   0   0]
 [  0 131   4   9   7   3]
 [  0   3 389   4   0   1]
 [  0   5   2 397   1   1]
 [  1   3   0   3 346  10]
 [  0   2   0   2   5 364]]
INFO:tensorflow:Validation accuracy = 96.43% (N=1847)
INFO:tensorflow:set_size=1948
INFO:tensorflow:Confusion Matrix:
 [[163   0   0   0   0   0]
 [  0 139   5   9   0  10]
 [  0   6 408   5   0   0]
 [  0   6   1 396   0   2]
 [  0   7   1   1 374  13]
 [  0   8   0   4   5 385]]
INFO:tensorflow:Test accuracy = 95.74% (N=1948)

**fused test:**
`python test.py --model_architecture ds_cnn --model_size_info 5 64 10 4 2 2 64 3 3 1 1 64 3 3 1 1 64 3 3 1 1 64 3 3 1 1 --dct_coefficient_count 10 --window_size_ms 40 --window_stride_ms 20 --data_url= --wanted_words=yes,no,on,off --checkpoint=work/DS_CNN/DS_CNN_S/training/best/ds_cnn_9631.ckpt-30400_bnfused`

> 2019-01-03 19:57:38.964112: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA
INFO:tensorflow:Restoring parameters from work/DS_CNN/DS_CNN_S/training/best/ds_cnn_9631.ckpt-30400_bnfused
INFO:tensorflow:set_size=14898
INFO:tensorflow:Confusion Matrix:
 [[   0 1241    0    0    0    0]
 [   0  589  712    0    0    0]
 [   0  409 2725    0    0    0]
 [   0  522 2566    0    0    0]
 [   0 2222  880    0    0    0]
 [   0 1594 1438    0    0    0]]
INFO:tensorflow:Training accuracy = 22.24% (N=14898)
INFO:tensorflow:set_size=1847
INFO:tensorflow:Confusion Matrix:
 [[  0 154   0   0   0   0]
 [  0  59  95   0   0   0]
 [  0  63 334   0   0   0]
 [  0  76 330   0   0   0]
 [  0 251 112   0   0   0]
 [  0 215 158   0   0   0]]
INFO:tensorflow:Validation accuracy = 21.28% (N=1847)
INFO:tensorflow:set_size=1948
INFO:tensorflow:Confusion Matrix:
 [[  0 163   0   0   0   0]
 [  0  64  99   0   0   0]
 [  0  51 368   0   0   0]
 [  0  78 327   0   0   0]
 [  0 279 117   0   0   0]
 [  0 208 194   0   0   0]]
INFO:tensorflow:Test accuracy = 22.18% (N=1948)

**Comment by yanyanem** [comment]
fused test: you should use quant_test.py to test ds_cnn_9631.ckpt-30400_bnfused.
